swimlane/core/resources/app_revision.py

```diff
@@ -0,0 +1,49 @@
+from swimlane.core.resources.app import App
+from swimlane.core.resources.revision_base import RevisionBase
+
+
+class AppRevision(RevisionBase):
+    """
+    Encapsulates a single revision returned from a History lookup.
+
+    Attributes:
+        Attributes:
+        modified_date: The date this app revision was created.
+        revision_number: The revision number of this app revision.
+        status: Indicates whether this revision is the current revision or a historical revision.
+        user: The user that saved this revision of the record.
+        version: The App corresponding to the data contained in this app revision.
+    """
+    def __init__(self, swimlane, raw):
+        super(AppRevision, self).__init__(swimlane, raw)
+
+    @staticmethod
+    def __separator():
+        """
+        Separator for unique ids. Unlikely to be found in application ids. Although technically we do not currently
+        validate app ids in the backend for specific characters so this sequence could be found.
+        """
+        return ' --- '
```

Comment:
could just convert to constant attribute at root of class, don't see any reason to have it be a method.